cve_bin_tool/checkers/irssi.py

```diff
@@ -20,7 +20,7 @@ class IrssiChecker(Checker):
         # r"# %%s : must be second - use Irssi; use Irssi::Irc; etc\.\.",
         # r"# When irssi expands the templates in \"format\", the final string would be:",
         # r"# When irssi sees this kind of text, it goes to find \"name\" from abstracts",
-        # r"Log file \{hilight $0\} is locked, probably by another running Irssi",
+        # r"Log file \{highlight $0\} is locked, probably by another running Irssi",
```

Comment:
Please remove this change from this PR.  It's not relevant and I'm not even sure it would be correct, since "hilight" may be the way it's written in the actual signature of irssi.